src/Common/tests/System/Net/Configuration.Security.cs

```diff
@@ -22,6 +22,10 @@ public static partial class Security
 
             public static Uri NegotiateServer => GetUriValue("COREFX_NET_SECURITY_NEGOSERVERURI");
 
+            public static Uri NegotiateClient => GetUriValue("COREFX_NET_SECURITY_NEGOCLIENTURI");
+
+            public static string NegotiateClientUser => GetValue("COREFX_NET_SECURITY_NEGOCLIENTUSER");
+
             public static string TlsRenegotiationServer => GetValue("COREFX_NET_SECURITY_TLSREGOTIATIONSERVER", "corefx-net-tls.azurewebsites.net");
```

Comment:
There's a typo in the existing value name. TLSREGOTI.. -> TLSRENEGOTI...

Reply:
Leaving this for now, it needs to be updated in tandem with the test infrastructure.